src/hooks/useCds/index.js

```diff
@@ -5,6 +5,7 @@ import { cdsResources } from 'services/fhir';
 import { valueSetJson } from 'services/valuesets';
 import { translateResponse, translateToggleChange } from './translate';
 import { stridesData } from './strides';
+import { logMsg } from 'util/logger';
```

Comment:
When I try to test while the logger is running, I see the `Logging at https://localhost:4040/log` console output, but then I immediately get the following error:
`Uncaught ReferenceError: fs is not defined`

Reply:
I also see this error when I run this branch of the dashboard while the logger is not running.

Reply:
I assumed we would need `cert` and `key` on client end, turned out we won't need them. Removed that part https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/pull/82/commits/431276833491c06f897c5d344e10d56f0ccbcb0b

Reply:
Got it, I an not longer getting that error now, and the Dashboard is running again without crashing.